collector/Dockerfile

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM otel/opentelemetry-collector-contrib:0.87.0
+FROM otel/opentelemetry-collector-contrib:0.100.0
```

Comment:
is this relevant to the service account command change? If not, have you verified that all of the configs are still valid with this version of the collector?

We do need to update this but it should probably be a separate PR because it could break the provided configs, so those all need to be tested

Reply:
Hi @damemi,

You are correct! This should be separate another PR. :)